### DISCUSSION TITLE
[@types/react-image-gallery] Missing 'loading' and 'thumbnailLoading' from type definition of 'type ReactImageGalleryItem'

### DISCUSSION THREAD

**Discussion by nik32** [DefinitelyTyped/DefinitelyTyped#62493] [Issues with a @types package]
**Describe the bug**
@adamwpc, @WToa The TS type definition of 'type ReactImageGalleryItem' does not contain definition for attributes 'loading' and 'thumbnailLoading' - even though they are valid properties for 'item' object according to the docs

**Image Gallery Version**
"react-image-gallery": "1.2.9"
"@types/react-image-gallery": "1.0.5",

**To Reproduce**
1. Set-up a TS-react app
2. Install the TS types for react-image-gallery
3. In VS code - just define the items array with type as 'ReactImageGalleryItem' and define the 'loading' or 'thumbnailLoading' property in this items array (as shown in the below image) -
<img width="983" alt="image" src="https://user-images.githubusercontent.com/51018572/193396937-b89aa5de-5d41-4dbc-a948-c32dd03f0950.png">

**Expected behavior**
TS should not produce error while using 'loading' or 'thumbnailLoading'. Just add these properties in the TS types

[a comment by an automated account is not shown]

**Comment by Semigradsky** [comment]
I have created PR: https://github.com/DefinitelyTyped/DefinitelyTyped/pull/62539
